diff --git a/build/index.js b/build/index.js
index 60e87ef01a3082850e8911d9a99488b30c478732..f9be3823146156257c04bec84836503c7bf9d7e6 100644
--- a/build/index.js
+++ b/build/index.js
@@ -1337,11 +1337,11 @@ async function processAuthorizationCodeOpenIDResponse(as, client, response, expe
     }
     if (expectedNonce === expectNoNonce) {
         if (claims.nonce !== undefined) {
-            throw OPE('unexpected ID Token "nonce" claim value', JWT_CLAIM_COMPARISON, {
-                expected: undefined,
-                claims,
-                claim: 'nonce',
-            });
+            // throw OPE('unexpected ID Token "nonce" claim value', JWT_CLAIM_COMPARISON, {
+            //     expected: undefined,
+            //     claims,
+            //     claim: 'nonce',
+            // });
         }
     }
     else if (claims.nonce !== expectedNonce) {
